src/core/Form/HintText/HintText.md

```diff
@@ -0,0 +1,8 @@
+HintText is a component that generates an accessible hint text for an adjacent element. Give the ID of the HintText as an `aria-describedby` value to the related component. Applicable form components in the library already have HintText as an integral part.
+
+```js
+import { HintText } from 'suomifi-ui-components';
+import React from 'react';
+
+<HintText>This is a hint regarding an adjacent input</HintText>;
+```
```

Comment:
Is span surely right html-tag for it?

Reply:
Span is the most versatile option here as we cannot be sure of the needed semantics when this is used. Good question though and we could probably use something else as well at least internally. Might be different tag for different components though. But I propose we leave it as it is for now.